roles/tackle/tasks/main.yml

```diff
@@ -885,6 +885,22 @@
     - (pathfinder_delete_db_volume|bool)
     - (pathfinder_pod.resources|length) == 0
 
+- name: "Remove RHSSO Subscription"
+  k8s:
+    state: absent
+    api_version: operators.coreos.com/v1alpha1
+    kind: Subscription
+    label_selectors:
+    - operators.coreos.com/rhsso-operator.openshift-mta =
+
+- name: "Remove RHSSO ClusterServiceVersion"
+  k8s:
+    state: absent
+    api_version: operators.coreos.com/v1alpha1
+    kind: ClusterServiceVersion
+    label_selectors:
+    - operators.coreos.com/rhsso-operator.openshift-mta =
+
```

Comment:
Is this ok here?
- indentation?
- no value for the equal?

Reply:
indetation adjusted to appease the linter.